package.json

```diff
@@ -33,7 +33,7 @@
     "shelljs": "^0.8.4"
   },
   "peerDepencencies": {
-    "eslint": ">=4.15.0 <6.0.0"
+    "eslint": ">=4.15.0"
```

Comment:
Can we specify `<8.0.0` please? This is in case 8.0.0 has a breaking change for plugin consumers.

Also, it might be good to ensure the devDependency for ESLint is at latest, so we can use CI to make sure the plugin won't have problems. Thanks!

Reply:
Fixed! Thanks for the explanation.